app/models/phone.rb

```diff
@@ -3,6 +3,10 @@ class Phone < ApplicationRecord
   
   validates :number, presence: true, length: {minimum: 9}
 
+  phony_normalize :number, default_country_code: 'IN'
```

Comment:
What happens if we have an international patient with a non-Indian number? Not a P0 feature but I want to think this through regardless.